src/base/static/components/input-form/index.js

```diff
@@ -298,7 +293,14 @@ class InputForm extends Component {
 
     let attrs = {
       ...this.state.fields
-        .filter(state => !!state.get("value"))
+        .filter(
+          state => {
+            return (
+              state.get("config").get("type") !== "lng_lat" &&
+              !!state.get("value")
+            );
+          },
+        )
```

Comment:
I'll see how much farther I can get with porting existing components to TS in this PR; I just want to be able to deliver this feature first thing Monday if possible.

Reply:
Sounds good - especially now that I know we can import TS modules into JS modules. Perhaps one of our Babel upgrades made that possible 🤔 

My main concern is that there is a lot of tech debt here, and I want us to be wise about adding to much code on top of an existing anti-pattern. And we'll want to port these over to TS before integrating with the new form backend.

But it makes sense if you want to punt this and get the feature shipped.

Reply:
I think since the change here is effectively just a single line it might be more streamlined to port this separately. I think porting this module will be a bear, though I'm happy to take it on next.

Reply:
Good point. Given that it's just a few lines, it's probably worth the ROI to accept the risk of regressions/complexity overhead and get the feature shipped sooner :+1: